locales/es.yaml

```diff
@@ -56,3 +56,4 @@ es:
   continue: continuar
   error_amount: Por favor, seleccione un valor que no sea menor que {{ min_amount }}.
   donate_now_amount: Donar {{ currency_symbol }}{ amount } ahora
+  other_ways_to_give:
```

Comment:
My goal here was to hide this text on the localized pages (as the places they link too are not localized). Is this a bad way to do that?